### PR TITLE
Add GH Action to ensure prettier as a presubmit job

### PR DESCRIPTION
**Pull request by rquitales**
For every PR/push made to the `main` branch, we should run a presubmit job to ensure that files are properly formatted with prettier. This is in addition to pre-commit hook work carried out in #77 as pre-commit hooks heavily relies on developer setup which we can't ensure is identical for all contributors.

This PR also contains commits that prettifies existing files to ensure we're starting from a good state.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -8,7 +8,8 @@
 
 .pac-container {
   z-index: 1501;
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New", monospace;
```
Comment (oeed) [inline src/App.css:11]: Do we want to have the prettier changes done on a separate PR, rather than merging them in to this one?

Comment (rquitales) [reply src/App.css:11]: I think it's better to have them squashed into 1 PR so we can prevent drifting in case other PRs get submitted in between the prettier change PR, and GH action PR. The changes are in separate commits anyway, which aren't squashed so when we review commit history, it doesn't really matter.

 Happy to go either way. LMKWYT

Comment (oeed) [reply src/App.css:11]: Fair argument, what do you think @noway

**Comment by noway** [comment]
@rquitales it would be hard to merge just at the right time without merge conflicts. Can you please only include the GH actions? 

We will have a failing GH action, but it's ok.

Then we atomically find a good time to prettify the whole codebase when everyone is asleep or something

**Comment by rquitales** [comment]
@noway Have updated the PR. Will let you merge once all files have been prettified.

**Review by noway — APPROVED** [review]
I think this should be good, full steam ahead!
